test/test_prototype_transforms_functional.py

```diff
@@ -505,6 +471,41 @@ def test_eager_vs_scripted(functional_info, sample_input):
     torch.testing.assert_close(eager, scripted)
 
 
+@pytest.mark.parametrize(
+    ("functional_info", "sample_input"),
+    [
+        pytest.param(
+            functional_info,
+            sample_input,
+            id=f"{functional_info.name}-{idx}",
+            marks=[
+                *(
+                    [pytest.mark.xfail(strict=False)]
+                    if functional_info.name
+                    in {
+                        "rotate_bounding_box",
+                        "crop_bounding_box",
+                        "resized_crop_bounding_box",
+                        "perspective_bounding_box",
+                        "elastic_bounding_box",
+                        "center_crop_bounding_box",
+                    }
+                    else []
+                )
+            ],
+        )
+        for functional_info in FUNCTIONAL_INFOS
+        for idx, sample_input in enumerate(functional_info.sample_inputs())
+    ],
+)
+def test_dtype_consistency(functional_info, sample_input):
```

Comment:
This test was also added on the old framework, because we haven't ported all kernels yet, but we need to make sure the behavior is consistent for the whole API.